modules/datastore/src/DataDictionary/AlterTableQuery/MySQLQuery.php

```diff
@@ -4,6 +4,7 @@
 
 use Drupal\Core\Database\StatementInterface;
 
+use Drupal\datastore_mysql_import\Service\MysqlImport;
```

Comment:
This module is not enabled by default.

Reply:
OR maybe the sanitizing/truncating of things should be in Drupal\datastore\Plugin\QueueWorker\ImportJob.php

Reply:
Moved sanitizing/truncating methods to ImportJob.